include/bluetooth/hci.h

```diff
@@ -973,12 +973,12 @@ struct bt_hci_rp_le_read_phy {
 	u8_t  rx_phy;
 } __packed;
 
-#define BT_HCI_LE_PHY_TX_ANY                    0x00
-#define BT_HCI_LE_PHY_RX_ANY                    0x01
+#define BT_HCI_LE_PHY_TX_ANY                    0x01
```

Comment:
The core spec talks about bit numbers, so it might be more intuitive to do BIT(0), etc here for a clearer mapping.